pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "cognite-transformations-cli"
-version = "2.1.2"
```

Comment:
this is forgotten but this version has already a bug due to cognite-sdk, so i don't bother releasing 2.1.3 in pyPI

Reply:
we could add a pre-commit check to make sure the versions coincide (like what exists now on python sdk).... not for this change, but worth to look at it

Reply:
makes sense, I'll create a task.

Reply:
https://cognitedata.atlassian.net/browse/CDF-16518